owid/catalog/meta.py

```diff
@@ -128,6 +141,27 @@ def version(self) -> Optional[str]:
 
         return None
 
+    def update_from_yaml(self, path: Union[Path, str]) -> None:
+        """Update metadata from a YAML file."""
```

Comment:
Doc: "The main reason for wanting to do this is to manually override what goes into Grapher before an export."